### ISSUE TITLE
question about different version of code.

### ISSUE THREAD

**Issue by ljjcoder** [TorchSSL/TorchSSL#58]
Thanks for your excellent work. I found that the results in cifar100 are different in different versions of paper. I noticed that your code also changed five months ago. I downloaded your code on 2021.10.19 and experimented on it. Is the code of 2021.10.19  different from the current one? Do I need to redo the experiment on the current code?

**Comment by qianlanwyd** [comment]
Yes. We fix the sampling bugs and update the results both in Arxiv and Neurips. Previous code will sample m times the labeled data if using m gpus.(The comparison was still fair but the results were not accurate. That’s why we update the results.). You need to rerun your experiments.
